components/packages/pb-input/src/pb-input.ts

```diff
@@ -0,0 +1,190 @@
+import {css, customElement, html, LitElement, property} from 'lit-element';
+// @ts-ignore
+import bootstrapStyle from './style.scss';
+// import {get, listenForLangChanged, registerTranslateConfig, use} from "lit-translate";
+import {listenForLangChanged, registerTranslateConfig, use} from "lit-translate";
+import * as i18n_en from "./i18n/en.json";
+import * as i18n_es from "./i18n/es-ES.json";
+import * as i18n_fr from "./i18n/fr.json";
+import * as i18n_ja from "./i18n/ja.json";
+import * as i18n_pt from "./i18n/pt-BR.json";
+
+// Registers i18n loader
+registerTranslateConfig({
+  loader: (lang) => Promise.resolve(PbInput.getCatalog(lang))
+});
+
+@customElement('pb-input')
+export class PbInput extends LitElement {
+
+  private name = "pbInput";
+
+  @property({ attribute: 'lang', type: String, reflect: true })
+  lang: string = "en";
+
+  // Common properties below are handled by the div above pb-input:
+
+  // @property({ attribute: 'width', type: String, reflect: true })
+  // private width: string = "12";
+  //
+  // @property({ attribute: 'css-classes', type: String, reflect: true })
+  // private cssClasses: string = "";
+  //
+  // @property({ attribute: 'hidden', type: Boolean, reflect: true })
+  // private hidden: boolean = false;
+
+  @property({ attribute: 'required', type: Boolean, reflect: true })
+  private required: boolean = false;
+
+  @property({ attribute: 'min-length', type: String, reflect: true })
+  private minLength: number = 0;
+
+  @property({ attribute: 'max-length', type: String, reflect: true })
+  private maxLength: number = Number.MAX_VALUE;
+
+  @property({ attribute: 'readonly', type: Boolean, reflect: true })
+  private readOnly: boolean = false;
+
+  @property({ attribute: 'label-hidden', type: Boolean, reflect: true })
+  private labelHidden: boolean = false;
+
+  @property({ attribute: 'label', type: String, reflect: true })
+  private label: string = "Default label";
+
+  @property({ attribute: 'label-position', type: String, reflect: true })
+  private labelPosition: string = "top";
```

Comment:
When i change this value, nothing occurs

Reply:
Yes indeed, need to check this